recipes/oneagent-sdk/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "oneagent-sdk" %}
+{% set version = "1.4.0.20210127.165413" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/oneagent-sdk-{{ version }}.zip
+  sha256: c76ca9f4aace3286f405ea4d5719967d48fe0a59d1c87f74309931001d0af57f
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - wheel
+  run:
+    - python !=3.0,!=3.1,!=3.2,!=3.3
```

Keep this simple, we don't support anything `<3.6` on conda-forge.
```suggestion
    - python >=3.6
```

---

It was generated by grayskull this way, but I agree that the simple version is nicer.